file-mod/cmd/replace.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/chef/ci-studio-common/lib"
```

Comment:
[note] So, in go-land, I believe "library" is not really a thing; commonly, you'd talk about packages. It's just a convention, but if you wanted to split up the `lib` package, there could be a `"github.com/chef/ci-studio-common/pkg/lines"` package exporting the `FindAndReplace` function. This would avoid the kitchensink "lib", I suppose -- but it's also not a terribly important thing.

[nit] It's often desired to sort and cluster imports like
```go
import (
	// stdlib ones
	"fmt"
	// ... etc

	// third party
	"github.com/spf13/cobra"
	// ... etc

	// internal deps
	"github.com/chef/ci-studio-common/lib"
)
```
`goimports` has a flag, `-local`, to help with this